### PR TITLE
Update First Timers Only Template

### PR DESCRIPTION
**Pull request by colindaviddick**
Fixes issue #806

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt jasmine`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] PR body includes `fixes #0000`-style reference to original issue #
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by TildaDares** [comment]
Hi @colindaviddick, can you change the PR title to something more descriptive? You could use the title of the issue this PR is resolving.

Also, can you link to the issue by adding `fixes #0000`? Thanks!

File: .github/first-timers-issue-template.md [hunk 1]
```diff
@@ -13,7 +13,12 @@ Nothing. This issue is meant to welcome you to Open Source :) We are happy to wa
 ### 📋 Step by Step
 
 - [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
-
+<details>
+  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
+  <h3>Claiming an issue</h3>
+  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
+  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
+</details>
```
Comment (TildaDares) [inline .github/first-timers-issue-template.md:21]:
```suggestion
</details>

```

@colindaviddick Can you add one empty line of empty space here?

Comment (colindaviddick) [reply .github/first-timers-issue-template.md:21]: Ok, done!

**Review by TildaDares — APPROVED** [review]
Thanks @colindaviddick. Amazing work!

**Pull request merged**